app/models/crate.js

```diff
@@ -130,8 +147,19 @@ export default class Crate extends Model {
     return [...(teams ?? []), ...(users ?? [])];
   });
 
-  loadVersionsTask = task(async () => {
-    return (await this.versions) ?? [];
+  /**
+   * @param {Object} [Options]
+   * @param {bool} [Options.reload]
+   * @param {bool} [Options.withReleaseTracks] - default: true if it has not yet been loaded else false
+   **/
+  loadVersionsTask = task(async ({ reload = false, withReleaseTracks } = {}) => {
+    let versionsRef = this.hasMany('versions');
+    let opts = { adapterOptions: { withReleaseTracks } };
+    if (opts.adapterOptions.withReleaseTracks == null) {
+      opts.adapterOptions.withReleaseTracks = this?.versions_meta?.release_tracks == null;
+    }
+    let fut = reload === true ? versionsRef.reload(opts) : versionsRef.load(opts);
+    return (await fut) ?? [];
   });
```

Comment:
I think we could also consider extending this to two tasks: a parent task to calculate the options, and a child task to perform the actual API request.

The reason for this is that for `reload`, we likely want to request with the same arguments. However, with a single task and no other state to hold the calculated options (withReleaseTracks), the value might change. If a child task is introduced, we could use `last.args` as calculated options.